data/src/test/java/org/apache/iceberg/data/TestGenericRecord.java

```diff
@@ -35,7 +34,7 @@ public void testGetNullValue() {
     GenericRecord record = GenericRecord.create(schema);
     record.set(0, null);
 
-    Assert.assertNull(record.get(0, type.typeId().javaClass()));
+    Assertions.assertThat(record.get(0, type.typeId().javaClass())).isNull();
```

Comment:
as previously mentioned, please use the static import of this method

Reply:
this hasn't been addressed yet

Reply:
@igoradulian can you also please address this and use a static import for `assertThat()`?